dspace/config/dspace.cfg

```diff
@@ -1624,6 +1624,33 @@ google.recaptcha.site-verify = https://www.google.com/recaptcha/api/siteverify
 # checkbox - The "I'm not a robot" Checkbox requires the user to click a checkbox indicating the user is not a robot.
 #google.recaptcha.mode =
 
+#------------------------------------------------------------------#
+#---------------REGISTRATION DATA CONFIGURATION--------------------#
+#------------------------------------------------------------------#
+
+# Configuration for the duration of the token depending on the type
+# the format used should be compatible with the standard DURATION format,
+# but without the prefix `PT`:
```

Comment:
Is there a link we can provider where the "standard DURATION format" is defined?  I'm only finding [ISO 8601](https://www.digi.com/resources/documentation/digidocs/90001488-13/reference/r_iso_8601_duration_format.htm), but this doesn't seem to be that format.  This implies to me that we need to add a link here, if possible.